CHANGELOG.md

```diff
@@ -1,3 +1,21 @@
+# 3.0.0
```

Comment:
I wonder if we add some sort of comment here that specifies that we don't have a V3 out yet?  I'm fine if we don't, but just thought I'd get the idea out there.